### ISSUE TITLE
Did not detect S9 not connected

### ISSUE THREAD

**Issue by carlcbilodeau** [anselal/antminer-monitor#72]
Something weard happended to me yesterday. Antminer-monitor was showing no problem for all machines. But looking at my pool dashboard I realized that one of them was not listed in the active workers.  It has been disconnected for 3 hours without me knowing.

Normally I trust antminer-monitor and I don't look at the pool dashboard. I did a warm-boot of machine number 18 and everything was ok after that.

I forgot to take a printscreen of the antminer-monitor screen. 

QUESTION: How could antminer-monitor detect a machine with a good status but that is not really working?

**Comment by anselal** [comment]
As i understand it the software was showing the miner as active without any errors, right? But you say that the miner was shiwing as inactive in your pool dashboard. Could it be that the miner switched to the second pool ? Maybe i can show the time past from the last share and notify if that goes beyond a threshold.

**Comment by carlcbilodeau** [comment]
I was on Antpool, all my S9 on BTC and there is no auto-switch on Antpool. I guess your solution about the last share should be right.
It was my second day at Antpool. I was previously on slushpool but was making 4$US/day for 15 days after elect per S9. With Antpool I am making 10$.

**Comment by anselal** [comment]
The miner switches automatically to the next pool if there is a connection error with the current pool. This feature is hardcoded into the miner's software.

PS: 10$ is way low unless it is your profit minus electricity

**Comment by carlcbilodeau** [comment]
You are right. It simply switch to my 2nd pool. I just saw it on the dashboard of the other pool. Did not think about this yesterday.
Could we have different color when it is mining on second or third pool? What do you think?
Yes 10$ is profit. I am totally lost on pools. I suppose I should go like you said : viabtc.

**Comment by anselal** [comment]
The good thing is that the software is working without problems.

I can add a field in the table with the current pool but the table will grow. You can always set a different worker name for every pool and check it since i display the worker name.

**Comment by carlcbilodeau** [comment]
Personnaly, I would raise an error in pink just like you do in the following exemple:

[ERROR] ASIC chips are missing from miner '10.0.0.xxx'. Your Antminer 'S9' has 'xxx/189 chips'.

It could be [WARNING] One or more ASICS are actives on alternate pool.

I would do it because I look at the screen 20 times a day. If we have a message on top, we will not miss it.

**Comment by anselal** [comment]
Or it could be just an [INFO]. I will look into it

**Comment by anselal** [comment]
@carlcbilodeau I fixed the bug that always showed the first miner and now it always shows the active miner.
The bug was fixed with at 1035377. See #118 
Are you OK with that ? Can I close this issue ?
